apps/financial-aid/backend/migrations/20220323111210-add-staff-municipality_ids-column.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface
+        .addColumn(
+          'staff',
+          'municipality_ids',
+          {
+            type: Sequelize.ARRAY(Sequelize.STRING),
+            allowNull: true,
+          },
+          { transaction: t },
+        )
+        .then(() =>
+          queryInterface.sequelize.query(
+            'UPDATE "staff"\
+            SET "municipality_ids" = ARRAY["municipality_id"]',
+            { transaction: t },
+          ),
+        )
+        .then(() =>
+        Promise.all([
+          queryInterface.removeColumn('staff', 'municipality_id', {
+            transaction: t,
+          }),
+          queryInterface.changeColumn(
```

Comment:
Would it be possible to remove this changeColumn query and instead in the addColumn query above have allowNull: false and defaultValue: []
?